src/length_encode.rs

```diff
@@ -73,13 +73,12 @@ where
 }
 
 /// Run-length encodes the lengths of the values in `lengths` according to the deflate
-/// specification. This is used for writing the code lengths for the huffman tables for
+/// specification. This is used for writing the code lengths for the Huffman tables for
 /// the deflate stream.
 ///
-/// Returns a tuple containing a vec of the encoded lengths
```

Comment:
Since the method has a unit return type, I suppose this comment was outdated.

Reply:
Indeed